tools/lkl/lib/virtio_net_tap.c

```diff
@@ -122,5 +196,24 @@ struct lkl_netdev *lkl_netdev_tap_create(const char *ifname)
 		return NULL;
 	}
 
+	#ifdef CONFIG_EVENTFD
```

Comment:
Using CONFIG_EVENTFD seems very hacky. What if I want lkl_sys_eventfd and to close the virtio? We are calling all kinds of host sys calls in this file. Doesn't it look fragile?